app.py

```diff
@@ -1,23 +1,56 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import os
+import requests
+from todo_item import ToDoItem
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    items = session.get_items()
+    board_id = os.getenv('BOARD_ID')
+    api_key = os.getenv('TRELLO_API_KEY')
+    api_secret = os.getenv('TRELLO_API_TOKEN')
+    response = requests.get(f'https://api.trello.com/1/boards/{board_id}/cards', params={'key': api_key, 'token': api_secret})
+    
+    trello_card_list = response.json()
+
+    our_card_list = []
+    for card in trello_card_list:
+        status = "Not Started" if card["idList"] == '5f51689cc895b228ffcd80e6' else "Completed"
+        our_card_list.append({ "name": card["name"], "id": card["id"], "status": status })
```

Comment:
You should be using ToDoItem on this line.

Instead of creating a dictionary (i.e. `{ ... }`), you should create a ToDoItem, (i.e. `ToDoItem(card["id"], status, card["name"])` 

Only thing you need to be careful of is your class currently sets self.title while your index.html expects item.name.
Those will need to match.